urlgrabber/grabber.py

```diff
@@ -657,7 +657,11 @@ def _init_default_logger(logspec=None):
     try:
         if logspec is None:
             logspec = os.environ['URLGRABBER_DEBUG']
-        dbinfo = logspec.split(',')
+        if "," in logspec:
+            dbinfo = logspec.split(',')
+        if isinstance(logspec, list):
+            dbinfo = logspec
```

Comment:
Maybe better this way?
```suggestion
        if isinstance(logspec, list):
            dbinfo = logspec
        elif "," in logspec:
            dbinfo = logspec.split(",")
```

Reply:
agree, same result but more clean. Thanks for the suggestion